### DISCUSSION TITLE
GraphQL Request Header

### DISCUSSION THREAD

**Discussion by uPagge** [quarkusio/quarkus#28712] [Q&A] [answered]
tell me how to get headers in GraphQL? I found this way, but maybe there is a better one
<img width="394" alt="Снимок экрана 2022-10-20 в 12 29 36" src="https://user-images.githubusercontent.com/40397740/196911656-9dbe45e2-b9bf-4711-a759-304f9aa3cc41.png">

[a comment by an automated account is not shown]

**Reply by jmartisk** [reply]
No, I don't think there is a better way right now. SmallRye GraphQL doesn't expose any specific API for this. Potentially we could put the headers into the `SmallRyeContext` if there's major demand, but I don't see it very useful when you can use the `CurrentVertxRequest`

**Reply by uPagge** [reply]
Ok, thanks for the prompt reply.
